operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapSplitter.java

```diff
@@ -22,8 +22,10 @@
   // The limit for a Kubernetes Config Map is 1MB, including all components of the map. We use a data limit a bit
   // below that to ensure that the map structures, including the keys, metadata and the results of JSON encoding, don't
   // accidentally put us over the limit.
-  @SuppressWarnings({"FieldMayBeFinal", "FieldCanBeLocal"}) // not private or local so that unit tests can set it.
-  private static int DATA_LIMIT = 900_000;
```

Comment:
What the heck? Interesting.

Reply:
https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html#:~:text=In%20Java%20SE%207%20and,the%20readability%20of%20your%20code.

Reply:
Personally not a fan of that feature. I can see why you undid it.

Reply:
This one wasn't intentional... I didn't know about the underscore feature and I thought I'd accidentally put that typo in the number value.

Reply:
At a glance, how long does it take you to recognize how large the number is each way? There is a reason that we use commas in long numbers, and the underscore feature provides the same benefit.

Reply:
So, could you please put it back...?

Reply:
Sure

Reply:
I really don't have anything vested in this - just found it mildly interesting. But personally, I would prefer NOT to have underscores. They make searching for a particular constant using 'grep' much harder, and I don't find that they significantly aid readability.

Reply:
I agree, Tom. That said, my removal of the underscore was unintentional and is not related to the intent of this PR. I'm going to revert to including the underscore for now.